src/index.scss

```diff
@@ -1,9 +1,22 @@
+// Feature flags
+$css--font-face: true;
+$css--plex: true;
 $feature-flags: (
-  ui-shell: true,
   grid-columns-16: true,
```

Comment:
This is missing in the tutorial, otherwise the footer is formatted wrong.
![image](https://user-images.githubusercontent.com/3808948/60533090-0ec69680-9cff-11e9-866e-164317358bd4.png)

Reply:
Created issue: https://github.com/carbon-design-system/carbon-tutorial/issues/1448